tests/dragonfly/replication_test.py

```diff
@@ -1987,8 +1987,9 @@ async def test_client_pause_with_replica(df_factory, df_seeder_factory):
     stats_after_sleep = await c_master.info("CommandStats")
     # Check no commands are executed except info and replconf called from replica
     for cmd, cmd_stats in stats_after_sleep.items():
-        if "cmdstat_info" != cmd and "cmdstat_replconf" != cmd_stats:
-            assert stats[cmd] == cmd_stats, cmd
+        if cmd in ["cmdstat_info", "cmdstat_replconf", "cmdstat_multi"]:
```

Comment:
python question and I really don't care for applying this. Can we use `not in` in this expression ? Would it work?

Reply:
we could, but I think this is more readable :)

Reply:
as I said I don't really care, I was just curious :stuck_out_tongue: